File: app/src/main/java/com/babestudios/companieshouse/search/pojos/CompanySearchResult.java
package com.babestudios.companieshouse.search.pojos;

import java.util.ArrayList;
import java.util.List;

import com.google.gson.annotations.SerializedName;

public class CompanySearchResult {

	@SerializedName("total_results")
	public Integer totalResults;

	@SerializedName("items")
	public List<CompanySearchResultItem> items = new ArrayList<CompanySearchResultItem>();

}

File: domain/src/main/java/com/babestudios/companyinfouk/domain/model/search/CompanySearchResult.kt
package com.babestudios.companyinfouk.domain.model.search

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class CompanySearchResult (

	@SerialName("total_results")
	var totalResults: Int = 0,

	@SerialName("items")
	var items: List<CompanySearchResultItem> = emptyList()

)
